### PR TITLE
Add info badge

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Screenshot from 2020-02-17 14-18-14

### CODE REVIEW

File: packages/design-tokens/properties/theme/fallbacks.json [hunk 1]
```diff
@@ -13,6 +13,10 @@
                 "base": {"value": "#14C684"},
                 "inverted": {"value": "#14C684"}
             },
+            "info": {
```
Comment (GeorgiSTodorov) [inline packages/design-tokens/properties/theme/fallbacks.json]: FInding I needed to add "info" here was very much non-straight forward for me.

**Comment by GeorgiSTodorov** [comment]
`npm ERR! 429 Too Many Requests - GET https://registry.npmjs.org/@babel/plugin-proposal-object-rest-spread/-/plugin-proposal-object-rest-spread-7.4.3.tgz`
Weird.

File: packages/design-tokens/properties/theme/fallbacks.json (hunk 1 above)
Comment (mperrotti) [reply packages/design-tokens/properties/theme/fallbacks.json]: Thanks, that's good feedback. Where would you expect to add or edit theme defaults?

**Comment by mperrotti** [comment]
Closing this because it's not the badge style being referred to in https://jira.d2iq.com/browse/D2IQ-63820.

I apologize for the confusion and wasted time.

**Pull request closed without merging**